data-reconciliation-app/test/test.sh

```diff
@@ -85,70 +98,71 @@ done
 # otherwise you can get permission issues.
 cd "${certificate_dir}"
 
-# -------------------------- Test cases --------------------------
-echo "Test start"
-
-printf "\n  -------- Test Ingestion Service --------  \n\n"
-
-ingestUrl="$server/app/ingest";
-
-memberName="member0"
-check_eq "$memberName - data ingest should succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json" $only_status_code)"
+ingestUrl="$server/app/ingest"
+reportUrl="$server/app/report"
```

Comment:
My latest changes may have introduced a conflict for your changes. 
feel free to each me in case you need help tidying it up.

Reply:
All sorted